src/librustc_trans/callee.rs

```diff
@@ -179,5 +180,8 @@ pub fn resolve_and_get_fn<'a, 'tcx>(ccx: &CrateContext<'a, 'tcx>,
                                     substs: &'tcx Substs<'tcx>)
                                     -> ValueRef
 {
-    get_fn(ccx, monomorphize::resolve(ccx.tcx(), def_id, substs))
+    get_fn(ccx, ty::Instance::resolve(ccx.tcx(),
+    ty::ParamEnv::empty(traits::Reveal::All),
+    def_id,
+    substs).unwrap())
```

Comment:
The indnetation here is off - it should be at least once to the right for the 3 above lines.